src/components/Notification/NotificationCard.tsx

```
@@ -90,23 +108,25 @@ export default function NotificationCard({
+                className={cn(
+                  matchedStatus && statusColorMap[matchedStatus].text,
+                )}
+              >
+                {text}
+              </span>
+            );
+          })}
+        </p>
 
-      <p className='mt-4 text-xs text-gray-600'>{relativeTime(createdAt)}</p>
+        <p className='mt-4 text-xs text-gray-600'>{relativeTime(createdAt)}</p>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)